src/ui/components/DialogShare/DialogShare.tsx

```diff
@@ -75,20 +82,30 @@ export const DialogShare: React.FC<DialogShareProps> = ({
     urlIdPrefix,
     initialParams = {},
     hasDefaultSize,
+    showEmbedLink = true,
+    showHideMenu = true,
+    showSelectorsCheckbox,
+    showFederation,
+    showCopyAndExitBtn,
 }) => {
     const [currentUrl, setCurrentUrl] = React.useState(
         getInitialLink(loadedData, propsData, urlIdPrefix, initialParams),
     );
     const [selectedTheme, setSelectedTheme] = React.useState('');
     const [selectedLang, setSelectedLang] = React.useState('');
+    const [isFederationSelected, setIsFederationSelected] = React.useState(Boolean(showFederation));
     const [hideMenu, setHideMenu] = React.useState(
         Boolean(initialParams[COMMON_URL_OPTIONS.NO_CONTROLS]),
     );
     const [hideComments, setHideComments] = React.useState(
         Boolean(initialParams[CHARTKIT_URL_OPTIONS.HIDE_COMMENTS]),
     );
+    const [isSelectorsSaved, setIsSelectorsSaved] = React.useState(
+        Boolean(initialParams[CHARTKIT_URL_OPTIONS.HIDE_COMMENTS]),
```

Comment:
Why do you use CHARTKIT_URL_OPTIONS.HIDE_COMMENTS value for initial isSelectorsSaved value?

Reply:
Changed to `false`